### ISSUE TITLE
OC8: External Shares not handled correctly.

### ISSUE THREAD

**Issue by ravermeister** [shibby/owncloud-youtubedl#7]
Hi, 
when you select an External Share as Download target, 
the File is not created at the "real" external location,
but in the "relative" location under the user directory

e.g.:
lets say my ownclod root dir is:  */path/to/oc_root*

my config for external share:
*Musik* -> */path/to/external/Musik*
and I have a subfolder inside *Musik* called *youtube*
![External Share](http://www.rimkus.it/devel/owncloud/oc_yt_error1.png)

when I now select a subfolder of the external share in the youtube downloader dialog e.g.:
/Musik/youtube

the downloaded files will be created in a **new** folder in: 
*/path/to/oc_root/Musik/youtube*

thats wrong, the **correct path** would be */path/to/external/Musik/youtube*

**Comment by shibby** [comment]
Yep, youtube-dl downloads to user's folder. With this configuration, it cannot download to shared folder. 
We have to get if a folder external or not. 
Or, we can get full path of a given path. 
But i'm not ok with owncloud api. I think it is too much complicated and o cannot understand and use it.
So, if someone will not fix this issue, this will not closed by me :(

**Comment by ravermeister** [comment]
the galleryplus project has solved this issue, maybe we can get the codepart from there:
https://github.com/interfasys/galleryplus

or ask a Developer from this project
